packages/terra-cell-grid/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "terra-cell-grid",
+  "main": "lib/index.js",
+  "version": "0.0.0",
+  "description": "The terra-cell-grid is a structural row based layout for aligning column cells within 100% width of its parent",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-core.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-cell-grid",
+    "CellGrid",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-core/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-core#readme",
+  "peerDependencies": {
+    "react": "^16.8.5",
+    "react-dom": "^16.8.5"
+  },
+  "dependencies": {
+    "classnames": "^2.2.5",
+    "prop-types": "^15.5.8",
+    "terra-doc-template": "^2.17.0"
+  },
+  "scripts": {
+    "compile": "babel --root-mode upward src --out-dir lib --copy-files",
+    "lint": "npm run lint:js && npm run lint:scss",
+    "lint:js": "eslint --ext .js,.jsx . --ignore-path ../../.eslintignore",
+    "lint:scss": "stylelint src/**/*.scss",
+    "precompile": "rm -rf lib",
+    "test": "npm run jest && npm run wdio",
+    "jest": "jest --config ../../jestconfig.js",
+    "wdio": "cd ../.. && npx wdio ./packages/$npm_package_name/wdio.conf.js"
```

Comment:
This should be [`wdio-default`](https://github.com/cerner/terra-core/blob/master/packages/terra-alert/package.json#L46) to stay consistent with other packages.

Reply:
additionally you'll have to add the `wdio` and `wdio-fusion` commands.
https://github.com/cerner/terra-core/blob/master/packages/terra-avatar/package.json#L42-L44

Reply:
Updated.